app/javascript/components/modals/BugReportModal.tsx

```diff
@@ -75,7 +75,7 @@ export const BugReportModal = ({
   }, [open])
 
   return (
-    <Modal isOpen={open} onRequestClose={onClose} {...props}>
+    <Modal open={open} onClose={onClose} {...props}>
```

Comment:
This change is due to me having introduced our own `Modal` component that wraps the `react-modal` version.

Reply:
Nice. This'll make @kntsoriano happy!

Reply:
I hope so!